src/GafferImage/ImageWriter.cpp

```diff
@@ -706,10 +711,24 @@ ImageWriter::ImageWriter( const std::string &name )
 	addChild( new ImagePlug( "in" ) );
 	addChild( new StringPlug( "fileName" ) );
 	addChild( new StringPlug( "channels", Gaffer::Plug::In, "*" ) );
+
+	ColorSpacePtr colorSpace = new ColorSpace( "__colorSpace" );
```

Comment:
No need to do it now if you don't think it'd be useful. As you say, it'll be trivial to add later if necessary.